config/optional/block.block.umami_banner_recipes.yml

```diff
@@ -1,21 +1,31 @@
+uuid: null
 langcode: en
 status: true
 dependencies:
+  enforced:
+      module:
+        - demo_umami_content
```

Comment:
If this is provided by the install profile, which doesn't rely on the content module - how can this be imported when the profile is installed?

I think this should just live in the other module, and be marked for cleanup

Reply:
@larowlan The demo umami profile has \Drupal::service('module_installer')->install(['demo_umami_content'], TRUE); for installing the demo_umami_content module. So, it gets imported then.